x-pack/plugins/enterprise_search/public/applications/shared/result/result.scss

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+.resultField:nth-child(odd) {
+  background-color: $euiColorLightestShade;
+}
+
+.resultField {
+  padding: $euiSizeXS $euiSizeS;
+}
+
+.resultExpandColumn {
+  border-left: $euiBorderThin;
+  align-items: flex-start;
+  justify-content: center;
+  height: 100%;
+  padding: $euiSizeS;
```

Comment:
```suggestion
  padding: 0 0 0 $euiSizeS;
```

Reply:
This should make sure that the right `expand` column doesn't have extra padding at the top or the right